sim/druid/rake.go

```diff
@@ -16,10 +16,12 @@ func (druid *Druid) registerRakeSpell() {
 
 	mangleAura := core.MangleAura(druid.CurrentTarget)
 
+	t9bonus := core.TernaryInt(druid.HasT9FeralSetBonus(2), 1, 0)
+
 	druid.Rake = druid.RegisterSpell(core.SpellConfig{
 		ActionID:    actionID,
 		SpellSchool: core.SpellSchoolPhysical,
-		Flags:       core.SpellFlagMeleeMetrics,
+		Flags:       core.SpellFlagMeleeMetrics | core.SpellFlagIgnoreResists,
```

Comment:
I dont think physical spells can be resisted so that flag is probably not needed

Reply:
rake needs to ignore armor, which is what this flag looks to be for as well, unless im missing some other reason it would be ignoring armor